site/content/entry/yuzu-progress-report-mar-2022/index.md

```diff
@@ -0,0 +1,282 @@
+Intel and Nvidia hardware do offer support and enjoy a much reduced shader count, and the associated reduced stuttering when building their cache.
+CPU thread count will be critial here, at least until AMD adds official support for the extension.
+
+## Graphical changes and optimizations
+
+[bunnei](https://github.com/bunnei) has the highlight of this month with the {{< gh-hovercard "8050" "rewrite of the `NVFlinger` implementation" >}}, a part of `Project Y.F.C`.
+
+NVFlinger is the service in charge of presenting to the display, so any improvement in this area will bring more even frametimes, perceived as less stuttering even when the frame rate count stays at a solid 60 FPS.
+
+Our original implementation was basically guessing done back in 2018, it was lacking in *several* areas.
+For example, yuzu would block threads until the last frame was ready to be presented. 
+This obviously leads to undesired stuttering unrelated to system performance.
+
+The new implementation is on par with the Nintendo Switch, based on the [Android Open Source Project](https://source.android.com/).
+This means yuzu changes its licence from GPLv2+ to GPLv3+ to accommodate the Apache 2.0 licence AOSP uses.
+
+{{< single-title-imgs
+	"ABZU"
+	"./abzubug.mp4"
+	"./abzufix.mp4"
+>}}
+
+Games like `ABZU` and `DRAGON BALL FighterZ` improve considerably, but other games like `Xenoblade Chronicles 2` require fixes that are still in testing in `Project Y.F.C.`
+
+{{< single-title-imgs
+	"DRAGON BALL FighterZ"
+	"./dbzbug.mp4"
+	"./dbzfix.mp4"
+>}}
+
+Enjoy the smooth as butter gameplay!
+
+[asLody](https://github.com/asLody) has been working on optimizing the `LOP3-LUT` shader instruction with the objective of improving performance and helping with getting `Hades` to work. 
+Sadly, this first implementation caused some issues.
+
+[degasus](https://github.com/degasus), which you may know from *a certain* [Dolphin emulator](https://dolphin-emu.org/?nocr=true), {{< gh-hovercard "7989" "managed to implement a patch" >}}, which later turned into its own pull request, that solved the issues and achieved the same level of optimization.
+
+This is not enough to get Hades running, `Project Y.F.C` will take care of that in the near future, but it’s a small global performance boost that mostly benefits low end GPUs. 
+It also provides easier to read code when performing maintenance.
+
+We documented our problems with video decoding on Linux Flatpak builds in previous progress reports.
+A check was added to avoid such crashes, but in the process, Windows builds got stuck on CPU decoding!
+Users of low thread CPUs could immediately tell the difference during video playbacks.
+
+By simply {{< gh-hovercard "8066" "disabling this check" >}}, Windows builds can again enjoy the extra performance gained from decoding via GPU.
```

Comment:
```suggestion
By simply {{< gh-hovercard "8066" "disabling this check on Windows" >}}, Windows builds can again enjoy the extra performance gained from decoding via GPU.
```
or some other rewording to make it not seem like the check was removed and Linux Flatpack builds regressed.